content/chapters/compute/lab/support/wait-for-me/Makefile

```
@@ -1,12 +1,2 @@
-DC = dmd
-BINARY = wait_for_me_threads
-
-all: $(BINARY)
-
-%: %.d
-	$(DC) $(DFLAGS) $^ -of=$@
-
-clean:
-	-rm -f *.o $(BINARIES)
-
-.PHONY: all clean
+BINARIES = wait_for_me_threads
+include ../../../../../common/makefile/multiple.mk
```

```suggestion
BINARY = wait_for_me_threads
include ../../../../../common/makefile/single.mk
```
There is no actual difference here; this suggestion is just for the sake of consistency.